src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/Symbols.vb

```diff
@@ -931,7 +931,7 @@ Namespace Microsoft.VisualBasic.CompilerServices
             ' For a WinRT object, we want to treat members of it's collection interfaces as members of the object
             ' itself. So GetMembers calls here to find the member in all the collection interfaces that this object
             ' implements.
-            <UnconditionalSuppressMessage("ReflectionAnalysis", "IL2065:UnrecognizedReflectionPattern",
+            <SuppressMessage("ReflectionAnalysis", "IL2065:UnrecognizedReflectionPattern",
```

Comment:
This should be able to be removed, right? If not, we should update the Justification.

Reply:
Yes, it's why this PR is a draft (this is a foreach)